docs/src/systems/weakly_compressible_sph.md

```diff
@@ -49,16 +44,7 @@ Modules = [TrixiParticles]
 Pages = [joinpath("schemes", "fluid", "weakly_compressible_sph", "state_equations.jl")]
 ```
 
-## References
-- Robert H. Cole. "Underwater Explosions". Princeton University Press, 1948.
-- J. P. Morris, P. J. Fox, Y. Zhu
-  "Modeling Low Reynolds Number Incompressible Flows Using SPH ".
-  In: Journal of Computational Physics , Vol. 136, No. 1, pages 214--226.
-  [doi: 10.1006/jcph.1997.5776](https://doi.org/10.1006/jcph.1997.5776)
-- S. Adami, X. Y. Hu, N. A. Adams.
-  "A generalized wall boundary condition for smoothed particle hydrodynamics".
-  In: Journal of Computational Physics 231, 21 (2012), pages 7057–7075.
-  [doi: 10.1016/J.JCP.2012.05.005](https://doi.org/10.1016/J.JCP.2012.05.005)
+TODO cite Morris
```

Comment:
Morris also didn't appear in the text

Reply:
We can add the citation of this paper to the Morris viscosity model.

Reply:
I've already added the ref there.

Reply:
So what about this TODO now?

Reply:
I can remove it, since Morris is cited in the viscosity docs. Does anyone know why it is listed here?

Reply:
I think they mentioned that the speed of sound needs to be 10x the maximum velocity in order to limit density variations to 1%. That could be the reason why we (me?) added it.

Reply:
<img width="632" alt="grafik" src="https://github.com/user-attachments/assets/710db81d-2976-4565-a8c3-953bb3d28681">
I might have added it because of this.
We write this as well in the docs, but without reference:
"For higher Reynolds numbers, `exponent=7` is recommended, whereas at lower Reynolds
numbers `exponent=1` yields more accurate pressure estimates since pressure and
density are proportional."